src/excalidraw-app/data/firebase.ts

```diff
@@ -160,5 +161,12 @@ export const loadFromFirebase = async (
   const storedScene = doc.data() as FirebaseStoredScene;
   const ciphertext = storedScene.ciphertext.toUint8Array();
   const iv = storedScene.iv.toUint8Array();
-  return restoreElements(await decryptElements(roomKey, iv, ciphertext));
+
+  const elements = await decryptElements(roomKey, iv, ciphertext);
+
+  if (socket) {
+    firebaseSceneVersionCache.set(socket, getSceneVersion(elements));
+  }
```

Comment:
I don't see it before, did you fix something?

Reply:
This fixes a case where you load a room and then leave it immediately, so that it doesn't bother you with "unsaved data prompt"

Reply:
Btw, in-code comments are really useful for those. Whenever some line of code is not obvious what it's doing it's a good idea to add a comment.